source/fundamentals/collations.txt

```diff
@@ -4,5 +4,298 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to run
+operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+MongoDB sorts them in the following order using the default binary collation:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+MongoDB sorts them in the following order using the Canadian French collation:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Usage
+-----
+
+You can specify a collation when you create a new collection, view, or index. You can also
+define a collation for specific operations that support collation. 
+
+Specifying a collation on a new collection or view defines the default collation for
+any operations called on these objects. Use the ``CreateCollection()`` or ``CreateView()``
+method with an opts parameter to set the collation.
+
+Specifying a collation on a new index defines the sort order for operations that 
+use that index. To use the collation for the index, you must provide a matching collation for
+an operation. Then, use the index with that operation. On an ``Indexes`` object,
+use the ``CreateOne()`` method with an opts parameter to set the collation.
+
+Specifying a collation on an operation overrides any default collation previously
+defined for a collection. See the :manual:`MongoDB manual </reference/collation/#collation-document>`
+for a list of operations that support collation.
+
+Collation Parameters
+~~~~~~~~~~~~~~~~~~~~
+
+The collation object contains the following parameters:
+
+.. code-block:: go
+
+   type Collation struct {
+     Locale <string>
+     CaseLevel <bool>
+     CaseFirst <string>
+     Strength <int>
+     NumericOrdering <bool>
+     Alternate <string>
+     MaxVariable <string>
+     Normalization <bool>
+     Backwards <bool>
+   }
```

Comment:
I took out the full list of fields and just mentioned that "Locale" is required, and the others are optional (but linked to the full list). I also renamed this section to "Specify a Collation" rather than "Collation Fields," and I moved it to earlier in the page.